management/dbaas/xtra_db_cluster_test.go

```diff
@@ -0,0 +1,217 @@
+							CPUm:        1,
+							MemoryBytes: "64",
+						},
+					},
+					Pxc: &xtra_db_cluster.CreateXtraDBClusterParamsBodyParamsPxc{
+						ComputeResources: &xtra_db_cluster.CreateXtraDBClusterParamsBodyParamsPxcComputeResources{
+							CPUm:        1,
+							MemoryBytes: "64",
+						},
+					},
+				},
+			},
+		}
+		paramsSecondPXC.WithContext(pmmapitests.Context)
+		_, err = dbaasClient.Default.XtraDBCluster.CreateXtraDBCluster(&paramsSecondPXC)
+		assert.NoError(t, err)
+
+		listXtraDBClustersParamsParam := xtra_db_cluster.ListXtraDBClustersParams{
+			Body: xtra_db_cluster.ListXtraDBClustersBody{
+				KubernetesClusterName: kubernetesClusterName,
+			},
+		}
+		listXtraDBClustersParamsParam.WithContext(pmmapitests.Context)
+		xtraDBClusters, err := dbaasClient.Default.XtraDBCluster.ListXtraDBClusters(&listXtraDBClustersParamsParam)
+		assert.NoError(t, err)
+
+		for _, name := range []string{"first.pxc.test.percona.com", "second.pxc.test.percona.com"} {
+			foundPXC := false
+			for _, pxc := range xtraDBClusters.Payload.Clusters {
+				if name == pxc.Name {
+					foundPXC = true
+
+					break
+				}
+			}
+			assert.True(t, foundPXC, "Cannot find PXC with name %s in cluster list", name)
+		}
+
+		paramsUpdatePXC := xtra_db_cluster.UpdateXtraDBClusterParams{
+			Body: xtra_db_cluster.UpdateXtraDBClusterBody{
+				KubernetesClusterName: kubernetesClusterName,
+				Name:                  "second.pxc.test.percona.com",
+				Params: &xtra_db_cluster.UpdateXtraDBClusterParamsBodyParams{
+					ClusterSize: 2,
+					Proxysql: &xtra_db_cluster.UpdateXtraDBClusterParamsBodyParamsProxysql{
+						ComputeResources: &xtra_db_cluster.UpdateXtraDBClusterParamsBodyParamsProxysqlComputeResources{
+							CPUm:        2,
+							MemoryBytes: "128",
+						},
+					},
+					Pxc: &xtra_db_cluster.UpdateXtraDBClusterParamsBodyParamsPxc{
+						ComputeResources: &xtra_db_cluster.UpdateXtraDBClusterParamsBodyParamsPxcComputeResources{
+							CPUm:        2,
+							MemoryBytes: "128",
+						},
+					},
+				},
+			},
+		}
+
+		paramsUpdatePXC.WithContext(pmmapitests.Context)
+		_, err = dbaasClient.Default.XtraDBCluster.UpdateXtraDBCluster(&paramsUpdatePXC)
+		pmmapitests.AssertAPIErrorf(t, err, 501, codes.Unimplemented, `This method is not implemented yet.`)
+
+		for _, pxc := range xtraDBClusters.Payload.Clusters {
+			if pxc.Name == "" {
+				continue
+			}
+			deleteXtraDBClusterParamsParam := xtra_db_cluster.DeleteXtraDBClusterParams{
+				Body: xtra_db_cluster.DeleteXtraDBClusterBody{
+					KubernetesClusterName: kubernetesClusterName,
+					Name:                  pxc.Name,
+				},
+			}
+			deleteXtraDBClusterParamsParam.WithContext(pmmapitests.Context)
+			_, err := dbaasClient.Default.XtraDBCluster.DeleteXtraDBCluster(&deleteXtraDBClusterParamsParam)
+			assert.NoError(t, err)
+		}
+	})
+
+	t.Run("Create_XtraDBCluster_Invalid_Name", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("CreateXtraDBClusterEmptyName", func(t *testing.T) {
```
Camel case and snake case in the same string?

Reply:
It's just the name of tests. There's no convention about it. We already have a zoo here. BTW - changed.